From herton@mandriva.com.br  Tue Feb 17 18:06:39 2009
From: Herton Ronaldo Krzesinski <herton@mandriva.com.br>
Date: Fri, 13 Feb 2009 08:35:12 -0500
Subject: Staging: agnx: mac80211 hw config change flags
Cc: gregkh@suse.de, Herton Ronaldo Krzesinski <herton@mandriva.com.br>
Message-ID: <1234532113-14081-1-git-send-email-herton@mandriva.com.br>


Adapt agnx after commit "mac80211: introduce hw config change flags",
detected by following build warning:
drivers/staging/agnx/pci.c:426: warning: initialization from incompatible pointer type

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com.br>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/agnx/pci.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/agnx/pci.c
+++ b/drivers/staging/agnx/pci.c
@@ -286,10 +286,10 @@ static void agnx_stop(struct ieee80211_h
 	rings_free(priv);
 }
 
-static int agnx_config(struct ieee80211_hw *dev,
-		       struct ieee80211_conf *conf)
+static int agnx_config(struct ieee80211_hw *dev, u32 changed)
 {
 	struct agnx_priv *priv = dev->priv;
+	struct ieee80211_conf *conf = &dev->conf;
 	int channel = ieee80211_frequency_to_channel(conf->channel->center_freq);
 	AGNX_TRACE;
 
